src/data/ores/stone.js

```diff
@@ -0,0 +1,16 @@
+import { itemType } from '../../enums'
+
+const { freeze } = Object
+
+/**
+ * @property farmhand.module:items.stone
+ * @type {farmhand.item}
+ */
+export const stone = freeze({
+  description: 'A piece of a rock.',
```

Comment:
I'm on the fence with whether this should be "piece of **a** rock" or "piece of rock." I feel like both forms are valid but the latter seems to be a bit more common. It's not a necessary change, but @lstebner I just wanted to get your thoughts here?

Reply:
haha, i'm laughing because I actually changed it in my last commit. i started to think "is a rock a piece of a stone?" and then changed it, but it would match the others to simply be "piece of rock" (as you mentioned) so i'm fine with changing it (back)!